sim-cli/src/parsing.rs

```diff
@@ -123,6 +123,13 @@ impl Cli {
             ));
         }
 
+        if sim_params.sim_network.is_empty() && !sim_params.exclude.is_empty() {
```

Comment:
Should also check that `activity` + `exclude` aren't specified at the same time?